docs/docs-content/vm-management/vm-migration-assistant/create-vm-migration-assistant-profile.md

```diff
@@ -35,10 +33,10 @@ Follow these steps to create a new add-on profile that will be applied to your e
 6. Palette displays the YAML file in the editor at right. You can edit the YAML as needed. Review the following
    parameters and adjust to your requirements if needed.
 
-   | **Parameter**                                                      | **Description**                                                                                                                    | **Default Value**            |
-   | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
-   | `vm-migration-assistant-ui.console.service.console.type`           | Choose whether to use a `NodePort`, `LoadBalancer`, or an `Ingress` to expose the service console.                                 | `"LoadBalancer"`             |
-   | `vm-migration-assistant-ui.console.service.console.ingress.domain` | If using Ingress, specify the domain name that the Ingress resource will use to expose the VM Migration Assistant service console. | `console.127.0.0.1.sslip.io` |
+   | **Parameter**                                                      | **Description**                                                                                                                    | **Default Value**               |
+   | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------- | ------------------------------- |
+   | `vm-migration-assistant-ui.console.service.console.type`           | Choose whether to use a `NodePort`, `LoadBalancer`, or an `Ingress` to expose the service console.                                 | `"LoadBalancer"`                |
+   | `vm-migration-assistant-ui.console.service.console.ingress.domain` | If using Ingress, specify the domain name that the Ingress resource will use to expose the VM Migration Assistant service console. | `vm-migration.spectrocloud.dev` |
```

Comment:
@benradstone Should this be `console.127.0.0.1.sslip.io` ? I see the Pack README has that value. `spectrocloud.dev` is only applicable to use, so I wonder if that's something that was only used for development purposes.

Reply:
@karl-cardenas-coding This was a specific change requested by @cloudmaniac.

Reply:
`console.127.0.0.1.sslip.io` was used for dev. The latest version of the README (which should be pushed today or tomorrow) will have the correct example.